storage/badger/version_beacon.go

```diff
@@ -0,0 +1,41 @@
+package badger
+
+import (
+	"github.com/dgraph-io/badger/v2"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/storage/badger/operation"
+)
+
+type VersionBeacons struct {
+	db *badger.DB
+}
+
+var _ storage.VersionBeacons = (*VersionBeacons)(nil)
+
+func NewVersionBeacons(collector module.CacheMetrics, db *badger.DB) *VersionBeacons {
+
+	res := &VersionBeacons{
+		db: db,
+	}
+
+	return res
+}
+
+// Highest finds the highest flow.VersionBeacon but no higher than maxHeight.
+// Returns storage.ErrNotFound if version beacon exists at or below the given height.
+func (r *VersionBeacons) Highest(maxHeight uint64) (*flow.VersionBeacon, uint64, error) {
```

Comment:
In most other places the wording is: "latest" version beacon, not "highest". Might be nice to also have that here.

Reply:
Yes, but this is highest but not higher than. Latests doesn't have the same meaning in my opinion